src/main/java/bio/terra/cli/app/utils/VersionCheckUtils.java

```diff
@@ -5,55 +5,32 @@
 import bio.terra.cli.service.WorkspaceManagerService;
 import bio.terra.workspace.model.SystemVersion;
 import java.lang.module.ModuleDescriptor.Version;
-import java.time.Duration;
 import java.time.OffsetDateTime;
-import java.util.Optional;
 import javax.annotation.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class VersionCheckUtils {
   private static final Logger logger = LoggerFactory.getLogger(VersionCheckUtils.class);
-  private static final Duration VERSION_CHECK_INTERVAL = Duration.ofMinutes(30);
 
   /** Query Workspace Manager for the oldest supported */
   public static boolean isObsolete() {
-    if (checkIntervalElapsed()) {
-      // update last checked time in the context file
-      VersionCheck updatedVersionCheck = new VersionCheck(OffsetDateTime.now());
-      Context.setVersionCheck(updatedVersionCheck);
+    // update last checked time in the context file
+    VersionCheck updatedVersionCheck = new VersionCheck(OffsetDateTime.now());
+    Context.setVersionCheck(updatedVersionCheck);
```

Comment:
i don't think we need to set this time anymore since we are not checking the version based on the interval elapse.